src/store.ts

```diff
@@ -216,6 +217,7 @@ const createAppOnboardingData = (data: FirestoreOnboardingUserData): AppOnboardi
   college: data.colleges[0].acronym,
   major: data.majors.map(({ acronym }) => acronym),
   minor: data.minors.map(({ acronym }) => acronym),
+  grad: data.grad.map(({ acronym }) => acronym),
```

Comment:
Our current users might not have the `grad` field and so we get a TypeError. 
![Screen Shot 2021-05-15 at 1 23 56 PM](https://user-images.githubusercontent.com/54298311/118372662-cde96b00-b580-11eb-9b9c-6e70f7b8b4ec.png)

I would take a look at my comment on @willespencer's PR: https://github.com/cornell-dti/course-plan/pull/500#discussion_r630116250

If we merge Will's PR first and then yours, it should be ok because I believe he resolved this issue on his end.

Reply:
My concern with merging mine first is mine actually changes things like Onboarding, which will break things until everything is done, while this PR should theoretically not change anything on the front-end. Might be better to just copy my solution over? but not sure

Reply:
@willespencer yes, you have a good point. If @vaishnavi17 can copy your solution over and the staging site looks good, then we can merge hers in first!